src/test/java/core/basesyntax/ProductCalculatorTest.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import core.basesyntax.service.FileService;
+import dto.Order;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+import parsers.OrderParser;
+
+public class ProductCalculatorTest {
```

Comment:
let's rename (and maybe refactor) test classes so that we could clearly understand which service we are testing